STL in C++ course(bangla) added

khalid586 opened the pull request:
## Add resource


### STL is a very important part in C++. A bangla course has been added which has great quality.

###  Great quality. Easy to understand.

### Avaailable on youtube.

### For book lists, is it a book? For course lists, is it a course? etc.

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

eshellman commented on courses/free-courses-bn.md:
```diff
@@ -73,6 +73,7 @@
 Shikkhangon BD
 * [Object Oriented C++ \| Bangla Tutorial](https://www.youtube.com/playlist?list=PLy7uM3PHzMF1hnqhFGE4_A8qTUfFmZ_3y) - 
 Online School
+* [Standard Template Library in C++](https://youtube.com/playlist?list=PLgLCjVh3O6Sgux985GYG22xkFt9z9Sq0_) - LoveExtendsCode
```
Don't invent titles! Use the title given by the resource.

khalid586 commented:
@eshellman done!

khalid586 commented:
@eshellman  please review the changes and merge if everything's okay. Thank you.

eshellman commented:
didd you mean to add the competitive programming link to this PR? Either remove it or explain it.

khalid586 commented:
> didd you mean to add the competitive programming link to this PR? Either remove it or explain it.

I added them by mistake. Removed them. I hope you will be able to merge now.

eshellman commented on courses/free-courses-bn.md:
```diff
@@ -73,6 +73,7 @@
 Shikkhangon BD
 * [Object Oriented C++ \| Bangla Tutorial](https://www.youtube.com/playlist?list=PLy7uM3PHzMF1hnqhFGE4_A8qTUfFmZ_3y) - 
 Online School
+* [C++ STL (Bangla)](https://youtube.com/playlist?list=PLgLCjVh3O6Sgux985GYG22xkFt9z9Sq0_) - LoveExtendsCode
```
please put lists in alphabetical order

khalid586 replied:
@eshellman alphabetically sorted.

khalid586 commented:
@eshellman I have sorted alphabetically. Have a look at it. Thanks.

eshellman commented:
thanks
